src/profile/webid.ts

```diff
@@ -76,7 +76,9 @@ export function getAltProfileUrlAllFrom(
  * A WebID Profile may be any RDF resource on the Web, it doesn't have
  * to be a Solid resource. That is why, in order to expose a Solid-enabled part
  * of their profile, some WebID profiles link to a Profile Resource, which may
- * be a Solid resource.
+ * be a Solid resource. A WebID resource should be public, so `getProfileAll` will
```

Comment:
```suggestion
 * be a Solid resource. A WebID resource MUST be public, so `getProfileAll` will
```

I think this is what the spec says, but I might be wrong

Reply:
Spec doesn't say "MUST" nor "should", in fact, it no longer says anything about WebID's other than they're resources — i.e., it reintroduced as of September 10th the ability for WebID's to require authentication to read, by way of omitting to specify that they must be publicly readable.

Reply:
Arguably, https://www.w3.org/2005/Incubator/webid/spec/identity/ can be interpreted to mandate the WebID to be public (that's the worst sentence to say about a specification), so unless the Solid-WebID spec adds additional constraints, it sounds like a fair assumption that WebID profiles are public documents.